modules/apps/headless/headless-admin-taxonomy/headless-admin-taxonomy-impl/src/main/java/com/liferay/headless/admin/taxonomy/internal/resource/v1_0/TaxonomyCategoryResourceImpl.java

```diff
@@ -524,6 +561,23 @@ private long _getTotalCount(Long siteId) {
 		return _assetCategoryLocalService.dynamicQueryCount(dynamicQuery);
 	}
 
+	private long _getVocabularyId(
+			AssetCategory assetCategory, TaxonomyCategory taxonomyCategory)
+		throws Exception {
+
+		if ((taxonomyCategory.getTaxonomyVocabularyId() != null) &&
+			(taxonomyCategory.getTaxonomyVocabularyId() > 0)) {
+
+			AssetVocabulary newAssetVocabulary =
```

Comment:
@sergiojimcos this should be renamed to `existing` too.

Reply:
See https://github.com/liferay-headless/liferay-portal/pull/982/commits/8489abe12230eed539b1ee02fbb14e577f1fec12.